ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/BlockResult.java

```diff
@@ -198,6 +206,11 @@ public String getGasLimit() {
     return gasLimit;
   }
 
+  @JsonGetter(value = "gasTarget")
```

Comment:
Do we want to add tests for impacted RPC methods in the case of an eip1559 transaction ?

Reply:
I was thinking the same thing actually.  This PR is probably best to wait until next week to merge, so we can implement whatever the consensus is from the [london readiness call](https://twitter.com/trent_vanepps/status/1388153403432480769?s=20) where json-rpc format will be discussed (hopefully finalized)

Reply:
I added a serialization test for the added fields, and removed gasTarget in favor of gasLimit.  should be gtg